client/src/components/History/HistoryBadge/HistoryBadge.module.scss

```diff
@@ -4,8 +4,12 @@
     text-align: center;
     font-size: 0.75em;  /* ~10px design * 1.2 design-to-app ratio */
     border-radius: 3.6px;
-    background-color: #9d85ef;
+    background-color: var(--global-color-accent--normal);
     color: rgb(253, 241, 241);
     margin-left: auto;
     margin-right: 2em;
 }
+
+.disabled {
+    background-color: var(--global-color-accent--light);
```

Comment:
__Inaccuracy__: In the UI design, the background color does not change; it is the color of the link text that changes. (Compare [`Onboarding Failed.png`](https://jira.tacc.utexas.edu/secure/attachment/14598/Onboarding%20Failed.png) and [`Onboarding Complete.png`](https://jira.tacc.utexas.edu/secure/attachment/14597/Onboarding%20Complete.png).)

__Hint__: To disable links:

- __A.__ Do __not__ use hyperlink. You can use a `<span>` tag instead, when links are "disabled".
- __B.__ Remove the `href` attribute ([pure JavaScript solution](https://css-tricks.com/how-to-disable-links/#just-dont-do-it)).
    - _The removal of the `href` attribute, after React renders it (and ensure it stays up to date with state change), __might__ be as simple as adding a `useEffect` hook._
- ~~__C.__ [Add obscure CSS and caveat-solution-heavy JavaScript that catches all use cases](https://css-tricks.com/how-to-disable-links/#thats-not-enough-i-want-something-more-complex-so-that-i-can-look-smarter).~~